### ISSUE TITLE
Hi, I have a same issue.

### ISSUE THREAD

**Issue by JinukKwon** [rosinality/stylegan2-pytorch#63]
I try to use your code but have a problem that "No module named 'fused'"

There are no requirements about fused, but should I install fused module?

**Comment by rosinality** [comment]
fused is the name of custom cuda extensions in the op directory. You need to check you can compile cuda programs and pytorch extensions.

**Comment by walbermr** [comment]
@JinukKwon what s.o are you using? Have you fixed it? I'm facing the same problem.

**Comment by walbermr** [comment]
I found the following:

1. Your cuda version must be >= 10.1, otherwise it will not compile
2. In order to compile it in windows, you must install the visualstudio compiler, and run the visualstudio developer comand line. I am using the cmd with native x64 tools.
3. If you tried to compile it sometime, failed and is still giving compilation error, I found no other way rather than reinstalling pytorch to clear the compiler cache.

**Comment by bennymi** [comment]
@walbermr Could you please expand on this a bit more? I'm having problems getting this implementation to run on Windows 10. I have cuda version 10.2 installed and I downloaded Visual Studio 2017, but I'm not sure which Individual Components I need to add. 
I'm getting this error:

![stylegan2-pytorch-ninja-problem](https://user-images.githubusercontent.com/20195536/80083385-d7f39280-8555-11ea-972c-526d2c89726e.png)

**Comment by bennymi** [comment]
@walbermr Could you please expand on this a bit more? I'm having problems getting this implementation to run on Windows 10. I have cuda version 10.2 installed and I downloaded Visual Studio 2017, but I'm not sure which Individual Components I need to add.
I'm getting this error:
![stylegan2-pytorch-ninja-problem](https://user-images.githubusercontent.com/20195536/80084182-cf4f8c00-8556-11ea-9208-90a7667eb9fe.png)

**Comment by walbermr** [comment]
@bennymi, Installing the c++ component for windows should be fine. When you install visualstudio you also install the developer comand line tool, I used this terminal window, it adds to path some the microsoft compiler, you can find it using the search bar.

![image](https://user-images.githubusercontent.com/26147726/80112512-8f3cd900-8557-11ea-97c5-17e420eb9f22.png)

**Comment by bennymi** [comment]
@walbermr thank you for your response. I got it working now, but actually decided to dual boot Ubuntu. That seems to have automatically solved most of the issues.

**Comment by walbermr** [comment]
Indeed, I think this thread should be renamed to help others that needs to use windows.
